apisix/plugins/http-logger.lua

```diff
@@ -169,6 +174,23 @@ local function gen_log_format(metadata)
 end
 
 
+-- remove stale objects from the memory after timer expires
+local function remove_stale_objects(premature)
+    if premature then
+        return
+    end
+
+    for key, batch in ipairs(buffers) do
+        if #batch.entry_buffer.entries == 0 and #batch.batch_to_process == 0 then
+            core.log.debug("removing batch processor stale object, route id:", tostring(key))
```

Comment:
Better to use a high level log so that we can notice user. What about using `warn`?

Reply:
OK, this logic was copied from other x-logger.lua, i'd like to fix all of them in another PR.

Reply:
@spacewander Fixed it in http-logger firstly.